### PR TITLE
Setup GHA.

### PR DESCRIPTION
**Pull request by icemac**
- Add support for Python 3.7, 3.8 and 3.9.
- Drop support for Python 3.4 and 3.5.
- Lint the code.

### CODE REVIEW

**Comment by icemac** [comment]
Hm, the readme tells that Python 2 support was dropped and is no longer claimed. But the tests still run fine.

**Comment by icemac** [comment]
I added an option to `meta/config`to drop Python 2 support in https://github.com/zopefoundation/meta/pull/45 and used it here.

File: src/importchecker/tests/fixture/absimport.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-import sys
-import sys.stderr
+import sys  # noqa: F401 imported but unused
+import sys.stderr  # noqa: F401 imported but unused
```
Comment (mgedmin) [inline src/importchecker/tests/fixture/absimport.py]: Uh, could we tell flake8 not to check anything under `tests/fixture/`?

(I'm guessing not easily with the current metaconfiguration system we have.)

**Review by mgedmin — COMMENTED** [review]
Do we need to drop CI on 2.7 right now, if the tests still work?

Basically LGTM otherwise, but maybe you'll want to update the meta template to avoid double negatives first.

**Comment by icemac** [comment]
I fixed the double negation and added an option to add additional flake8 configuration so it is no longer necessary to change the test fixture files.

**Review by mgedmin — APPROVED** [review]
LGTM!

**Pull request merged**